### DISCUSSION TITLE
Support for non-string values

### DISCUSSION THREAD

**Discussion by nikhilgarg28** [alicebob/miniredis#247] [Q&A]
Does the miniredis only support string values or does it also support other types (e.g. []bytes, or structs etc.)? I'm assuming it's just strings but want to confirm.

**Comment by alicebob** [comment]
Hi,
thanks for your question!

generally your code would use an existing library to talk to (mini)redis, and that library can do whatever encoding it wants. But in the end (mini)redis stores values as bytes and doesn't touch or care what's in the value.

I might be misunderstanding the question, in which case a small code snippet might clear things up.
